cmd/ndm_daemonset/controller/controller.go

```diff
@@ -132,6 +137,13 @@ func NewController() (*Controller, error) {
 	}
 	controller.Namespace = ns
 
+	// get the namespace in which NDM is installed
+	labelList, err := getLabelList()
+	if err != nil {
+		return controller, err
+	}
+	controller.LabelList = labelList
```

Comment:
Why does this need to be stored in the controller struct?

Reply:
I thought since this struct is the implementation of disk resources and this env is related to bd resource itself and this env will be fixed initially so better to make it a part of this struct. But not a direct dependency or something. Can we removed if we do not want it to be a part of controller struct.

Reply:
Better to remove as now I feel this isn't necessary. Should I remove it?